packages/falter-berlin-tunnelmanager/files/up.sh

```diff
@@ -19,9 +19,81 @@ get_configured_ips() {
     ip -o -4 a s | awk -F ' |\/' '{print $7}'
 }
 
-get_ips_from_prefix "$prefix"
+cleanup_olsr_wg_config() {
+    # all_olsr_interfaces=$(echo "/int" | nc 127.0.0.1 2006 | tail +3 | awk '{print $1}')
+    olsr_wg_interfaces=$(echo "/int" | nc 127.0.0.1 2006 | grep wg_ | awk '{print $1}')
+
+    i=0
+    while uci get olsrd.@Interface[$i] &>/dev/null; do
```

Comment:
please consider using the uci-lib from `/lib/config/uci.sh`. This applies for several other places.

Reply:
There is nothing like that.

Reply:
As written in the matrix-caht, we will postpone that to a later point.